config/config.exs

```diff
@@ -67,6 +67,22 @@ config :spacesuit, :health_route, %{ path: "/health", enabled: true }
 # Do we call out to an external session service that can process JWT tokens?
 config :spacesuit, session_service: %{ enabled: false }
 
+# Configuration for the CORS middleware
+config :spacesuit, cors: %{
+  # Main kill switch to disable the middleware
+  enabled: false, 
+  # Required prefix match in order to process CORS
+  path_prefixes: ["/matched"],
```

Comment:
It should definitely be in the test config because the tests rely on it. But we need some kind of placeholder in the main config so that people can see how it's supposed to look when configured. If you have some default examples in there, you don't have to, for example, explain in text that the path should begin with a slash, and that it's an array. So, I copied over the values from the test because they are reasonably easy to understand. Since the plugin is disabled, it doesn't hurt to have them here. We could alternatively comment them all out here.